support/Pipelines/Bbh/Inspiral.yaml

```diff
@@ -378,7 +388,7 @@ ControlSystems:
       AverageTimescaleFraction: 0.25
       Average0thDeriv: false
     Controller:
-      UpdateFraction: 0.03
+      UpdateFraction: 0.3 #Changed from 0.03 to 0.3 to increase run speed
```

Comment:
Move these comments to their own line above `UpdateFraction` (same for the rest of the control systems)